README.md

```diff
@@ -1,5 +1,5 @@
 # Spring Boot MicroServices Template
-This repository is an example of how to get Microservices going using Spring Boot, Spring Cloud, Spring OAuth 2 and Netflix OSS frameworks.
+This repository is an example of how to get microservices going using Spring Boot, Spring Cloud, Spring OAuth 2 and Netflix OSS frameworks.
```

Comment:
Please consistently use "Microservices" using capital M to define a specific pattern as a noun.

Reply:
commit edited 👍